File: FizzBuzz.py
# User inputted number for FizzBuzz

userNum = int(input("Enter number: "))

if userNum % 3 == 0 and userNum % 5 == 0:
    print(userNum, "FizzBuzz")
elif userNum % 5 == 0:
    print(userNum, "Buzz")
elif userNum % 3 == 0:
    print(userNum, "Fizz")
else:
    print(userNum, "Neither Fizz nor Buzz")

# Iterate with a for loop

for num in range(1, 101):
    if num % 3 == 0 and num % 5 == 0:
        print(num, "FizzBuzz")
    elif num % 5 == 0:
        print(num, "Buzz")
    elif num % 3 == 0:
        print(num, "Fizz")
    else:
        print(num, "Neither Fizz nor Buzz")

File: FizzBuzz.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // User inputted number for FizzBuzz

        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter number: ");
        int number = scanner.nextInt();
        scanner.close();

        if (number % 3 == 0 && number % 5 == 0) {
            System.out.println(number + " FizzBuzz");
        } else if (number % 3 == 0) {
            System.out.println(number + " Fizz");
        } else if (number % 5 == 0) {
            System.out.println(number + " Buzz");
        } else {
            System.out.println(number + " Neither Fizz nor Buzz");
        }

        // Iterate with a for loop

        for (int i = 1; i < 101; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println(i + " FizzBuzz");
            } else if (i % 3 == 0) {
                System.out.println(i + " Fizz");
            } else if (i % 5 == 0) {
                System.out.println(i + " Buzz");
            } else {
                System.out.println(i + " Neither Fizz nor Buzz");
            }
        }
    }
}
